service/telemetry/config.go

```diff
@@ -104,6 +104,18 @@ type LogsSamplingConfig struct {
 	Thereafter int `mapstructure:"thereafter"`
 }
 
+// MetricReader exposes configuration of metric readers to end users.
+// TODO: replace this temporary struct w/ auto-generated struct from jsonschema
+//
+// Experimental: *NOTE* this structure is subject to change or removal in the future.
```

Comment:
Is there anything from the configuration SIG we can link to from here?

Reply:
Added a link to the configuration repo, note the follow up PR adds the generated code